docs/pages/setup/reference/cli.mdx

```diff
@@ -743,6 +745,58 @@ add_headers:
       "Authorization": "Bearer tokentokentoken"
 ```
 
+#### Aliases
+
+Aliases allow you to define custom commands or change the default flag values for existing commands.
```

Comment:
I think it would be helpful to include a sentence about when/why you would use this.